src/parser.c

```diff
@@ -2872,6 +2875,10 @@ void parse_internal()
     type->base_type = TYPE_int;
     type->size = 4;
 
+    type = add_named_type("_Bool");
```

Comment:
Here, add comments to explain the `_Bool` type as defined in the C99 specification, for developers who may not be familiar with it.